### PR TITLE
fix: add property of clearAriaLabel in phoneInput for combobox

### PR DESCRIPTION
**Pull request by neelabhnagaich**
## Description
We have added clearAriaLabel to our phoneInput component and passed it combobox element for getting the localized message or label from the target source.

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/phone-input/PhoneInput.ts [hunk 1]
```diff
@@ -227,6 +228,7 @@ export namespace PhoneInput {
             @change-selected="${(e: CustomEvent) => this.handleCountryChange(e)}"
             clear-icon-height="${this.clearIconHeight}"
             with-custom-content
+            clear-aria-label="${this.clearAriaLabel}"
```
Comment (merajcal) [inline web-components/src/components/phone-input/PhoneInput.ts:231]: what is clear-aria-label?

**Review by merajcal — CHANGES_REQUESTED** [review]

File: web-components/src/components/phone-input/PhoneInput.ts (hunk 1 above)
Comment (merajcal) [reply web-components/src/components/phone-input/PhoneInput.ts:231]: by default  add english value in  clear-aria-label if value is not passed by client code

Comment (neelabhnagaich) [reply web-components/src/components/phone-input/PhoneInput.ts:231]: > what is clear-aria-label?

clear-aria-label is the attribute that combobox is expecting. Also combobox has the default set for this attribute.
Will add the default value here too.

Comment (neelabhnagaich) [reply web-components/src/components/phone-input/PhoneInput.ts:231]: default value added

**Review by merajcal — APPROVED** [review]

**Pull request merged**